### PR TITLE
Change individual donor payment flow

### LINKED ISSUE #237: Website 'payment' flow
Access this file: https://opensourcelk.slack.com/files/UDF4HA4TZ/F03QPBA2MB6/donate_flow_change_22-jul-2022.pptx  for details on the flow:

Step 1: Go from an aid package on the home page to the detailed view of an aid package in the goal pending tab
Screen Shot 2022-07-23 at 09 38 56

Step 2: Click Donate button in the detailed aid package view 
https://donor-portal-medicinesfor-lk.vercel.app/package/9
Screen Shot 2022-07-23 at 09 40 34

Step 3: Get directed to the donors page (page is now called how to donate):
https://donor-portal-medicinesfor-lk.vercel.app/donors

How to donate

Step 4: Click on Individual Donors 

Step 5: View a page which allows individual donors to pay (page not created, content not provided yet). Please create page as a minimum, call it Medicines for Sri Lanka for now

### PR DESCRIPTION
**Pull request by dinuka-rp**
## Purpose
The purpose of this PR is to fix #237, #240 <issue-number>

## Goals

- Implements the new web payment flow for individual donors
- Donation amount input won't be displayed for users who aren't logged in

### Screenshots

https://user-images.githubusercontent.com/47103883/180818738-b2376b19-8876-46f6-bf0a-426115c81aa2.mov

### CODE REVIEW

**Comment by dinuka-rp** [comment]
Updated UI of the donors page
image

File: donor-portal/src/pages/donors/individual-donors/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import HeaderImage from "../../layout/header-image";
+import Page from "../../layout/page";
+import "./styles.css";
+
+export default function IndividualDonors() {
+  return (
+    <Page className="donors">
+      {/* <div>IndividualDonors</div> */}
```
Comment (anjula-sack) [inline donor-portal/src/pages/donors/individual-donors/index.tsx:9]: Let's remove comments

File: donor-portal/src/components/navbar/index.tsx [hunk 2]
```diff
@@ -22,14 +22,14 @@ export default function NavBar() {
         <div className="logo" />
       </Link>
       <div className="nav-links">
-        <NavLink to="/" className="nav-link" exact>
+        {/* <NavLink to="/" className="nav-link" exact>
```
Comment (anjula-sack) [inline donor-portal/src/components/navbar/index.tsx:25]: Why this one is commented?

File: donor-portal/src/pages/donors/index.tsx [hunk 3]
```diff
@@ -24,11 +24,11 @@ export default function Donors() {
         <InfoBox
           iconUrl="assets/images/charity2.jpg"
           text="Navigate to the login page to begin registration"
-          heading="Charities"
+          heading="Charities, Foundations and Corporations"
           link="/login"
           buttonText="Login"
         />
-        <InfoBox
+        {/* <InfoBox
```
Comment (anjula-sack) [inline donor-portal/src/pages/donors/index.tsx:31]: Is this needed?

File: donor-portal/src/components/navbar/index.tsx (hunk 2 above)
Comment (dinuka-rp) [reply donor-portal/src/components/navbar/index.tsx:25]: @anjula-sack it's not in the new UI

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done @dinuka-rp !

**Pull request merged**